src/features/toolbar/components/ToolbarContent/ToolbarContent.tsx

```
@@ -1,52 +1,66 @@
-
-                    {section.name}
-                    <ChevronDown16 className={s.sectionArrow} />
-                  </label>
-                </div>
-              }
-            </Fragment>
-          ),
-      )}
+      {toolbarContent.map(({ sectionName, buttons }, index) => (
+        <Fragment key={sectionName}>
+          {index > 0 && <div className={s.sectionDivider}></div>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)